### PR TITLE
Add requests feedback screen after reply to it

### PR DESCRIPTION
**Pull request by djcouto**
Demo:

https://user-images.githubusercontent.com/17494742/128217297-c3ca92e0-d472-4ae9-b638-c53484f1724f.mov

### CODE REVIEW

File: src/popup/components/Request/index.tsx [hunk 1]
```diff
@@ -397,4 +407,74 @@ function Requests(props: RequestsProps) {
...
+    <TopComponent>
+      <HeaderContainer>
+        <RequestIcon
+          requester={requester.icon}
+          status={RequestsStatus.ACCEPTED}
+        />
+      </HeaderContainer>
+      <TitleContainer>
+        <Title>
+          {requester.name} <AcceptedLabel>allowed</AcceptedLabel> to access your
```
Comment (frm) [inline src/popup/components/Request/index.tsx]: Can we change this to `was allowed`? The wording is a bit weird without it.

File: src/popup/components/Request/index.tsx [hunk 2]
```diff
@@ -397,4 +407,74 @@ function Requests(props: RequestsProps) {
...
+    <TopComponent>
+      <HeaderContainer>
+        <RequestIcon
+          requester={requester.icon}
+          status={RequestsStatus.DECLINED}
+        />
+      </HeaderContainer>
+      <TitleContainer>
+        <Title>
+          {requester.name} <DeclinedLabel>denied</DeclinedLabel> to access your
```
Comment (frm) [inline src/popup/components/Request/index.tsx]: Same as previous comment. `was denied`

**Review by frm — CHANGES_REQUESTED** [review]

File: src/popup/components/Request/index.tsx (hunk 1 above)
Comment (djcouto) [reply src/popup/components/Request/index.tsx]: Ok, fixed 👍

File: src/popup/components/Request/index.tsx (hunk 2 above)
Comment (djcouto) [reply src/popup/components/Request/index.tsx]: Ok, fixed 👍

**Review by frm — APPROVED** [review]

**Pull request merged**